### PR TITLE
Whorfnsignore

### PR DESCRIPTION
**Pull request by eurogig**
Made changes to ensure the admission controller doesn't block manifests from critical namespaces but also allows for customisation of what is a critical namespace via a new configMap.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: admissioncontroller/k8s/deployment.yaml [hunk 1]
```diff
@@ -59,7 +60,12 @@ spec:
         - name: BC_SOURCE
           value: admissionController
         - name: CKV_GITHUB_CONFIG_FETCH_DATA
-          value: "False"
+          value: "False"     
+        - name: DEBUG
+          valueFrom:
+            configMapKeyRef:
+              name: whorfconfig # The ConfigMap this value comes from.
+              key: debug        # Are we in debug mode?
```
Comment (schosterbarak) [inline admissioncontroller/k8s/deployment.yaml:68]: do we want to keep it in debug?

File: admissioncontroller/whorf.py [hunk 2]
```diff
@@ -31,10 +40,16 @@ def validating_webhook():
         webhook.logger.error('K8s UID failed security checks. Request rejected!')
         return admission_response(False, uid, response)
 
+    # check we're not in the kube-system namespace
+    namespace = request_info["request"].get("namespace")
+    if namespace in ignore_list:
+        print("Namespace in ignore list. Ignoring validation")
```
Comment (schosterbarak) [inline admissioncontroller/whorf.py:46]: use the logger over print

Comment (eurogig) [reply admissioncontroller/whorf.py:46]: Both are present.  Shall I delete print anyway?

File: admissioncontroller/k8s/deployment.yaml (hunk 1 above)
Comment (eurogig) [reply admissioncontroller/k8s/deployment.yaml:68]: Debug status is dictated by the config map which is by default 'no'

**Review by nimrodkor — APPROVED** [review]

**Pull request merged**